regexes/oss.yml

```diff
@@ -264,6 +263,13 @@
   name: 'FreeBSD'
   version: ''
 
+##########
+# Wear OS (https://wearos.google.com/)
+##########
+- regex: 'Chrome/.+Odd/'
```

Comment:
`Chrome/(\d+\.[.\d]+) Odd/`
maybe we should do a not too greedy regular-regex?

Reply:
https://en.wikipedia.org/wiki/Samsung_Galaxy_Watch_series

Samsung Galaxy Watch 4Galaxy Watch 4 Classic[5] | SM-R860 SM-R865 SM-R870 SM-R875 SM-R880 SM-R885 SM-R890 SM-R895 | August 27, 2021 | Exynos W920 dual core 1.18 GHz | 4G/LTE with eSIM (Galaxy Watch 4 LTE-Version only)Bluetooth 5.0 Wi-Fi 802.11 a/b/g/n 2.4+5GHz NFC A-GPS, GLONASS, Beidou, Galileo | 30.4mm Corning Gorilla Glass DX+ (360x360) (1.2")34.6mm Corning Gorilla Glass DX+ (360x360) (1.4") | WearOS (OS 3.0) | 1.5 GB RAM | 16 GB | 247 mAh (40mm)361 mAh (44mm)
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
Samsung Galaxy Watch 5Galaxy Watch 5 Pro[6] | SM-R905 (40mm)SM-R915 (44mm)SM-R925 (45mm) | August 26, 2022 | 4G/LTE with eSIM (Galaxy Watch 5/5 Pro LTE-Version only)Bluetooth 5.2 Wi-Fi 802.11 a/b/g/n 2.4+5GHz NFC GPS, Glonass, Beidou, Galileo | 30.4mm Sapphire crystal (396x396 Super AMOLED) (1.2") [40m]34.6mm Sapphire crystal (450x450 Super AMOLED) (1.4") [44m and Pro] | WearOS (OS 3.5) | 284 mAh (40mm)410 mAh (44mm)590 mAh (45mm Pro)


`Chrome/(\d+\.[.\d]+) Odd/|SM-R(?:8[6-9]|9)`